lib/LedgerNodePeers.js

```diff
@@ -0,0 +1,145 @@
+/*!
+ * Ledger node operations management class.
```

Comment:
not sure the name of the module should be in the copyright.

Reply:
Fixed.